test/ContentService.tests.js

```diff
@@ -299,12 +299,17 @@ define(function (require) {
                             "fieldDefinitionIdentifier": "title",
                             "languageCode": "eng-US",
                             "fieldValue": "This is a title"
+                        },
+                        requestCallbacks = {
+                            upload: {},
+                            onerror: function () {}
                         };
 
                     contentCreateStruct.body.ContentCreate.fields.field.push(fieldInfo);
 
                     contentService.createContent(
                         contentCreateStruct,
+                        requestCallbacks,
```

Comment:
actually you should leave this test as is to validate the BC and add a new test where createContent is called with its new optional parameter